server/gateways/cloudflare.js

```diff
@@ -0,0 +1,55 @@
+import config from 'config';
+import cloudflare from 'cloudflare';
+import { get } from 'lodash';
+
+import logger from '../lib/logger';
+
+// Load config
+const cfConfig = {
+  email: get(config, 'clouflare.email'),
+  key: get(config, 'clouflare.key'),
+};
+
+export const zone = get(config, 'clouflare.zone');
```

Comment:
Why should that be exported?

Reply:
Almost all queries from `cloudflare` library expect a `zone` to be passed in params

Reply:
But I'll remove the cloudflare export to only export the methods that will be used